content/en/developers/integrations/log_pipeline.md

```diff
@@ -16,111 +16,165 @@ further_reading:
-5. Click **Add**.
+**Note**: Define the [unit][11] of a measure based on what the attribute represents.
+5. To help navigate the facet list specify a facet **Group**. If one does not exist enter the name of the group matching the source tag and a description of the new group, and select **New group**.
+6. Click **Add** to create the facet.
 
-To help navigate the facet list, facets are grouped together. For fields specific to the integration logs, create a **single group with the same name** as the `source` tag. 
+#### Configure and edit facets
 
-1. In the log panel, click the Cog icon next to the attribute that you want in the new group.
+1. In the log panel, click the Cog icon next to the attribute that you want to configure or group.
 2. Select **Edit facet/measure for @attribute**. If there isn't a facet for the attribute yet, select **Create facet/measure for @attribute**.
-3. Click **Advanced options**.
-4. In the **Group** field, enter the name of the group matching the source tag and a description of the new group, and select **New group**.
-5. Click **Update**.
-
-**Guidelines**
-- Before creating a new facet for an integration, review if the attribute should be remapped to a [standard attribute][6] instead. Facets for standard attributes are added automatically by Datadog when the log pipeline is published.
-- Not all attributes are meant to be used as a facet. Non-faceted attributes are still searchable. The need for facets in integrations is focused on three things:
-1. Facets that are measures allow for associated units with an attribute. For example an attribute "response_time" could have a unit of "ms" or "s". 
-2. Facets provide a straightforward filtering interface for logs. Each facet is listed under the group heading and usable for filtering.
-3. Facets allow for attributes with low readability to be renamed with a label that is easier to understand. For example: @deviceCPUper → Device CPU Utilization Percentage
-
-**Requirements:**
-- Use standard attributes as much as possible.
-- All facets that are not mapped to reserved or standard attributes should be namespaced with the integration name.
-- A facet has a source. It can be `log` for attributes or `tag` for tags.
-- A facet has a type (String, Boolean, Double or Integer) which matches the type of the attribute. If the type of the value of the attribute does not match the one of the facet, the attribute is not indexed with the facet.
-- Double and Integer facets can have a unit. Units are composed of a family (such as time or bytes) and of a name (such as millisecond or gibibyte).
-- A facet is stored in groups and has a description.
-- If you remap an attribute and keep both, define a facet on a single one.
+3. Click **Add** or **Update** when done.
+
+**Tips**
+* Measures should have a unit where possible. Measures can be assigned a [unit][27]. Two families of units are available, `TIME` and `BYTES`, with units such as `millisecond` or `gibibyte`.
```

Comment:
should we tell to add processors to convert to these units if current data is in seconds or hours for example?

Reply:
There's a link to the document with the full set of units.